config/yosbr/config/relicex.json

```diff
@@ -1,15 +1,15 @@
 {
   "chestsHaveLoot": true,
-  "chestsHaveRelicChance": 15,
+  "chestsHaveRelicChance": 28,
   "chestsHaveLesserOrbChance": 8,
-  "chestsHaveGreaterOrbChance": 2,
-  "chestsHaveTomeChance": 7,
+  "chestsHaveGreaterOrbChance": 1,
+  "chestsHaveTomeChance": 0,
   "dropsOnlyFromPlayerKills": true,
   "dragonDropsStone": false,
-  "mobsDropLootChance": 3,
+  "mobsDropLootChance": 4,
   "mobDropIsRelicChance": 50,
```

Comment:
It is already too easy to get decent immortal gear. I got good relics within 15 minutes of playing.
We shouldn't change `"mobsDropLootChance"` and we should lower `"mobDropIsRelicChance"`.

Reply:
It's harder if you're starting at a low level, if you have high luckiness already you're gonna have an easy time getting a lot of relics. With the current config the base chance is basically 1.5% per mob? i do agree though, I'll make the relic chance lower.